modules/ocf/manifests/packages.pp

```diff
@@ -10,14 +10,14 @@
   # special snowflake packages that require some config
   include ocf::packages::git
   include ocf::packages::grub
-  include ocf::packages::memtest
-  include ocf::packages::microcode
   include ocf::packages::ntp
   include ocf::packages::rsync
   include ocf::packages::shell
-  include ocf::packages::smart
   include ocf::packages::ssh
 
+  # any packages from hiera
+  create_resources(package, hiera('packages'))
```

Comment:
Not a huge fan of `create_resources` as it's often abused. I do like that it at least makes it obvious that the data is coming from hiera though, that's nice.